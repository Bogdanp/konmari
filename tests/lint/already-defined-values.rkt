#lang racket/base

(define-values (a a b)
  (values 1 2 3))

(define-values (a _ _)
  (values 1 2 3))

(define-values (a _ _)
  (define-values (b b)
    (values 1 2))
  (values b b))

(define-values (x y z)
  (values 1 2 3))
