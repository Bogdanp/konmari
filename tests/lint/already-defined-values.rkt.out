/already-defined-values.rkt:3:19:error:identifier 'a' is already defined
/already-defined-values.rkt:6:17:error:identifier 'a' is already defined
/already-defined-values.rkt:6:21:error:identifier '_' is already defined
/already-defined-values.rkt:10:19:warning:identifier 'b' shadows an earlier binding
/already-defined-values.rkt:10:21:error:identifier 'b' is already defined
/already-defined-values.rkt:9:17:error:identifier 'a' is already defined
/already-defined-values.rkt:9:19:error:identifier '_' is already defined
/already-defined-values.rkt:9:21:error:identifier '_' is already defined
/already-defined-values.rkt:14:21:warning:identifier 'z' is never used
/already-defined-values.rkt:3:21:warning:identifier 'b' is never used
/already-defined-values.rkt:14:19:warning:identifier 'y' is never used
/already-defined-values.rkt:3:17:warning:identifier 'a' is never used
/already-defined-values.rkt:14:17:warning:identifier 'x' is never used
